ERCS/erc-6538.md

```diff
@@ -280,7 +292,7 @@ You can find an implementation of this standard above.
 
 ## Security Considerations
 
-In the event of a compromised private key, the registrant should promptly un-register from the stealth key registry to prevent loss of future funds sent to the compromised account. To do this without getting the ETH needed drained by bots, the registrant should use a private mempool service such as Flashbots or create the signature offline and submit it to the registry contract through a relayer.
+In the event of a compromised private key, the registrant should promptly un-register from the stealth key registry to prevent loss of future funds sent to the compromised account. To do this without getting the ETH for transaction fees drained by bots, the registrant should use a private mempool service or create the signature offline and submit it to the registry contract through a relayer.
```

Comment:
I think we can remove the last sentence here

Reply:
Removed in https://github.com/garyghayrat/ERCs/pull/5/commits/4a4a2f39258df0c9ce59cf94707f9ba99d446fec